Dockerfile.no-systemd

```diff
@@ -0,0 +1,36 @@
+FROM balenalib/amd64-debian:buster-run
+
+ENV TERM xterm
+
+# Install ops/sre related packages.
+RUN install_packages \
+		htop \
+		jq \
+		nano \
+		netcat \
+		net-tools \
+		ifupdown \
+		strace \
+		vim \
+		wget
+# And configure them.
+COPY src/htoprc /root/.config/htop/
+
+# We use NodeJs everywhere.
+ENV NODE_VERSION 10.16.3
+ENV NPM_VERSION 6.10.3
+RUN curl -SL "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.gz" | tar xz -C /usr/local --strip-components=1 \
+	&& npm install -g npm@"$NPM_VERSION" \
+	&& npm cache clear --force \
+	&& rm -rf /tmp/*
+
+# Confd binary will be removed once we migrate to a different configuration management.
+ENV CONFD_VERSION 0.16.0
```

Comment:
CONFD is expected to be updated less frequently. Moving its installation over line 20, will allow for this layer to be reuse, and optimize node/npm updates.

Reply:
Should be addressed.